projects/convenience.md

```diff
@@ -0,0 +1,74 @@
+# Convenience Project
+
+## Background and description
+
+Convenience as a SIG was initially proposed by Ted Young back in 2021. It
+was postponed to prioritize completion of signal specs and stabilization. A
+document was written at this time describing user research to do and a
+brainstorming session,
+https://docs.google.com/document/d/1bjiw5L4E8FztRtfsy4Nu9A6qSP-BJlK6BBzl6Rea-Ds/edit#heading=h.b7qckyjgu28i
```

Comment:
```suggestion
https://docs.google.com/document/d/1bjiw5L4E8FztRtfsy4Nu9A6qSP-BJlK6BBzl6Rea-Ds/edit#heading=h.b7qckyjgu28i
```

can you provide a tl;dr for this document?

Reply:
Yup, will do!

Reply:
Finally updated.